source/content/guides/wordpress-configurations/07-wordpress-font-library.md

```diff
@@ -0,0 +1,78 @@
+
+For more information, refer to [this Gutenberg issue](https://github.com/WordPress/gutenberg/issues/58696).
+
+</Alert>
+
+## Considerations
+
+WordPress handles fonts more like **media files** than **plugins or themes**. This means that when fonts are added to one Pantheon site environment, they will not necessarily exist in your other WordPress environments (e.g. installing a font on Dev does not mean it will push to Test when you deploy). In WordPress, fonts have two parts, there is an underlying font post type (similar to the `attachment` post type for media files) and the physical font files themselves. Without the font post type existing in the database, WordPress has no way of knowing that a font is installed (in the same way that WordPress has no way that a particular image exists in the `/uploads` directory if it was not uploaded via the media library).
+
+If you intend to override the Pantheon behavior, care should be taken that you are not doing so with the assumption that committing font files to your repository will make them available automatically on your other environments.
+
+## Troubleshooting
+
+### "No font faces were installed" error message
+
+![No font faces were installed](../../../images/wordpress-configurations/07-no-font-faces-installed.png)
+
+If you have disabled our modification or are uploading to a directory that is not writeable, you may see this error message. Ensure that if you are overriding our `font_dir` filter, that the directory you are using instead is writeable (e.g. in `wp-content/uploads/`).
+
+### I've cloned my database from Live and my font is installed but not "active"
+
+![Font is installed but not active](../../../images/wordpress-configurations/07-font-installed-not-active.png)
+
+If you've cloned your database and files from your live environment to Dev or Test and the font appears in your Font Library but is not "active" (it displays a message like "0/1 variants active"), you can click into the font, select the variant you want, and click update. 
+
+![Update Font](../../../images/wordpress-configurations/07-font-update.png)
+
+Whether or not a font is "active" is similar to whether or not a plugin or theme is active. If a font is not active, it means the font is installed and recognized by WordPress but it cannot be used on the site until you activate it.
+
+### I've uploaded fonts to my `/fonts` directory, but they aren't showing up in the Font Library
+
+Fonts need to be installed via the Font Library in the WordPress admin. This is because fonts contain data that is stored in a post type. Simply having a font in your `/fonts` directory does not mean it will be recognized by WordPress.
+
+## More Resources
+* [Font Library Gutenberg Tracking Ticket](https://github.com/WordPress/gutenberg/issues/55277)
```

Comment:
Are there any WordPress core docs that could be linked here?

Reply:
No, not yet. Otherwise they would be linked.